inference-engine/cmake/vpu_dependencies.cmake

```diff
@@ -6,14 +6,14 @@ include_guard(GLOBAL)
 
 set(VPU_SUPPORTED_FIRMWARES usb-ma2x8x pcie-ma2x8x)
 set(VPU_SUPPORTED_FIRMWARES_HASH
-    "420b300d193f7fcfe7e3f9bbec6c247d65b784a500b5cd2effb7cb1ec6e1b209"
-    "bfe3caf270b168b9de18ef88f04bde3907d7d12a679f1fa7cc580423c35db637")
+    "224fdb7356909d4381e82a3178f273effec368c782bcb9f8d9ba69d7429bb8ea"
+    "d5a9ecb1358c1f3c849b7998d2b361eb5604daeba46e21a98547d51a33b5c3e9")
 
 #
 # Default packages
 #
 
-set(FIRMWARE_PACKAGE_VERSION 1688)
+set(FIRMWARE_PACKAGE_VERSION 1706)
```

Comment:
As far as I know, we decided not to change the firmware, and the corresponding request was closed

Reply:
You are correct, I've removed those changes.